umap/static/umap/js/modules/global.js

```diff
@@ -1,6 +1,7 @@
 import URLs from './urls.js'
 import Browser from './browser.js'
 import Facets from './facets.js'
+import Rules from './rules.js'
```

Comment:
Just naming this "rules" might be too broad. Maybe "filtering_rules" ?

Reply:
I'd keep it simple, until we have another kind of rules (just like we have browser and not "data browser" for eg.).